### PR TITLE
Update cws-test: Integration and Unit Tests (skipped)

### PR DESCRIPTION
**Pull request by voxparcxls**
### Run test.sh

```
>$ ./test.sh
```

### Integration Tests

Results:

Screen Shot 2022-05-11 at 2 24 32 PM

IT Build:
Screen Shot 2022-05-11 at 2 26 48 PM

------------

### Unit Tests

Results:
Screen Shot 2022-05-11 at 2 28 17 PM

UT Build:

Screen Shot 2022-05-11 at 2 27 42 PM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,6 +135,16 @@ The above script will build CWS, verify your configuration, then will start the
 
 You can stop CWS by running `./stop_dev.sh`. The script will bring down the console and all local workers.
 
+### Running Unit and Integration Test: cws-test
+
+Start `test.sh` script by running:
+
+```
+bash -c './test.sh'
```
Comment (jamesfwood) [inline README.md:143]: Don't need bash -c.  That is only for Jenkins.

File: cws-test/src/test/resources/hostname.txt [hunk 2]
```diff
@@ -1 +1 @@
-localhost
+localhost
```
Comment (jamesfwood) [inline cws-test/src/test/resources/hostname.txt:1]: I think this should maybe be included in the .gitignore.  Delete this file from the repo.
Add it as:
hostname.txt

Comment (voxparcxls) [reply cws-test/src/test/resources/hostname.txt]: Yes. I also added:
jacoco-reports/
target/

to .gitignore

File: README.md (hunk 1 above)
Comment (voxparcxls) [reply README.md]: Fixed

File: .gitignore [hunk 3]
```diff
@@ -26,4 +26,8 @@ install/logging/logstash-*.zip
 *personal_dev.sh
 
 # CWS backups
-.backups/
\ No newline at end of file
+.backups/
+
+# Jacoco Reports
+jacoco-reports/
+target/
```
[the remark this answers is not in the document]
Comment (voxparcxls) [reply .gitignore]: Updated

**Review by jamesfwood — APPROVED** [review]

**Pull request merged**